package.json

```diff
@@ -3,21 +3,22 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "start": "NODE_ENV=dev node app.js",
-    "prod": "NODE_ENV=prod node app.js",
+    "start": "cross-env NODE_ENV=dev node app.js",
+    "prod": "cross-env NODE_ENV=prod node app.js",
```

Comment:
We can use different command name. Maybe change start to start-u for unix and start-w for windows.

Reply:
It seems that start-w can't work on my Win.

Reply:
As long as you change the words in the quote mark, you can try every word you want. It should work, e.g. "start-w": "NODE_ENV=dev node app.js". (Though I'm not sure about how they detect the special symbol in win)